scripts/setup_quickstart.sh

```diff
@@ -40,11 +40,17 @@ if check_secrets || [[ ${SAMPLE} == "installations" ]]; then
 
   git clone https://github.com/firebase/quickstart-ios.git
   $scripts_dir/localize_podfile.swift quickstart-ios/"$SAMPLE"/Podfile "$RELEASE_TESTING"
-  if [ ! -z "$RELEASE_TESTING" ]; then
+  if [ "$RELEASE_TESTING" == "nightly_release_testing" ]; then
```

Comment:
update the flag of nightly release testing to "nightly_release_testing" and the flag for release candidate to  "prerelease_testing". Prerelease testing will be set up in another pr.